Adyen/Analytics/AnalyticsProvider/TelemetryData.swift

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+internal struct TelemetryData: Encodable {
+
+    // MARK: - Properties
+
+    internal var version: String? {
+        Bundle(for: AnalyticsProvider.self).infoDictionary?["CFBundleShortVersionString"] as? String
+    }
+
+    internal let channel: String = "iOS"
+
+    internal var locale: String {
+        let languageCode = Locale.current.languageCode ?? ""
+        let regionCode = Locale.current.regionCode ?? ""
+        return "\(languageCode)_\(regionCode)"
+    }
+
+    internal let userAgent: String? = nil
+
+    internal var deviceBrand: String {
```

Comment:
Isn't `UIDevice.current.model` enough for our requirements? This seems a bit much

Reply:
`UIDevice.current.model` only returns values such as "iPhone", "iPad", "iPod", etc. and we'd like to have the specific model. That's why we're using that ugly logic xD